src/app/_components/profiles/PublicProfileHero.tsx

```
@@ -11,6 +11,7 @@ import {
   Text,
   Title,
 } from "@mantine/core";
+import { facebookUrl, instagramUrl, tikTokUrl, twitterUrl, vimeoUrl, youTubeUrl } from "~/_utils";
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)